src/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './item'
+require 'Date'
```

Comment:
Nice working your game class, kindly change the `require 'Date'` to `require 'date'`, its throws the error in the screenshot below when you use `require 'Date'`.
![--Date-Error](https://user-images.githubusercontent.com/48371156/216775814-86fec185-2588-4a70-87ae-e82941e1a3a0.png)

Reply:
We have changed `Date` to `date` also